src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -69,89 +69,88 @@ def handle_exception(ex):  # pylint: disable=too-many-locals, too-many-statement
     # Print the traceback and exception message
     logger.debug(traceback.format_exc())
 
-    with CommandLoggerContext(logger):
```

Comment:
`CommandLoggerContext` will be called again at:

https://github.com/Azure/azure-cli/blob/4560e9603d93560ededeaa209e807690d130ae9d/src/azure-cli-core/azure/cli/core/azclierror.py#L62-L65

Calling it here is redundant.

Reply:
Besides, no log is written here, so these lines shouldn't be surrounded by `CommandLoggerContext`.